### PR TITLE
Electrolyzer curve update

### PR DESCRIPTION
**Pull request by FJanssen-TNO**
add factor to determine the location on the specific power curve for the maximum efficiency of the electrolyzer.

### CODE REVIEW

File: src/mesido/esdl/esdl_heat_model.py [hunk 1]
```diff
@@ -1948,6 +1948,7 @@ def convert_electrolyzer(self, asset: Asset) -> Tuple[Type[Electrolyzer], MODIFI
         Optional ESDL fields:
         ---------------------
         - technicalLifetime
+        - powerFactor
```
Comment (KobusVanRooyen) [inline src/mesido/esdl/esdl_heat_model.py:1951]: Update the CHANGELOG

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO review has been completed

File: src/mesido/esdl/esdl_heat_model.py (hunk 1 above)
Comment (FJanssen-TNO) [reply src/mesido/esdl/esdl_heat_model.py:1951]: Updated. Also added the ESDL example creation to the changelog.

**Review by KobusVanRooyen — APPROVED** [review]

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO it has been approved

**Pull request merged**